src/analytics/amplitude.py

```diff
@@ -124,3 +156,56 @@ def inner(*args, **kwargs):
         return res
 
     return inner
+
+
+def track_revenue_event(
+    user,
+    revenue_type: Literal["BOUNTY_FEE", "FUNDRAISE_CONTRIBUTION_FEE", "SUPPORT_FEE", "DOI_FEE", "WITHDRAWAL_FEE"],
+    rsc_revenue: str,
+    usd_revenue: str = None,
+    transaction_method: Literal["OFF_CHAIN", "ON_CHAIN", "STRIPE"] = None,
+
+    # it's useful to be able to see e.g. how much did we make on paper tips versus comment tips.
+    # so we should use these fields to point to the object that the revenue is associated with.
+    # and not simply the Purchase/Balance/Fundraise object.
+    content_type: str = None,
+    object_id: str = None,
+
+    additional_properties: dict = {},
+):
+    """
+    Helper function to track revenue events.
+    Performs the conversion from RSC to USD if usd_revenue is None.
+    """
+
+    amp = None
+    if DEVELOPMENT:
+        return
+
+    try:
+        if usd_revenue is None:
+            if not isinstance(rsc_revenue, float):
+                rsc_revenue_float = float(rsc_revenue)
+            else:
+                rsc_revenue_float = rsc_revenue
+
+            usd_revenue_float = RscExchangeRate.rsc_to_usd(rsc_revenue_float)
+            usd_revenue = str(usd_revenue_float)
+    except Exception as e:
+        log_info("Error converting RSC to USD", e)
```

Comment:
Wondering if we should log error to sentry here instead of info so we get notified in slack